webportal/config/webportal.config.js

```diff
@@ -27,7 +27,8 @@ let config = {
   restServerUri: process.env.REST_SERVER_URI,
   grafanaUri: process.env.GRAFANA_URI,
   k8sDashboardUri: process.env.K8S_DASHBOARD_URI,
-  k8sApiServerUri: process.env.K8S_API_SERVER_URI
+  k8sApiServerUri: process.env.K8S_API_SERVER_URI,
+  exportPort: process.env.EXPORT_PORT
```

Comment:
change a variable name? `exportPort` is confusing.

Reply:
Changed to exporterPort